tests/integ/modin/sql_counter.py

```diff
@@ -119,8 +119,10 @@ def __init__(
         high_count_reason=None,
         **kwargs,
     ) -> "SqlCounter":
+        from tests.integ.modin.conftest import SKIP_SQL_COUNT_CHECK
+
```

Comment:
@sfc-gh-lmukhopadhyay I thought you add a skip for stored sproc here also, where did it go? i want to unify this two changes so that we just use the same way to control skip of sql counter check

Reply:
I added it to [sql_count_checker decorator](https://github.com/snowflakedb/snowpark-python/pull/1721/files#diff-745c288d4dfae7e02324d3c313319c2a5757068eaa059e8a9dd08698e458b7d9R365) only, I was going to add it to the SqlCounter object instead today but it'd be better if we can reuse this

Reply:
yes, let's use the same thing. However, i am not sure if the change here will work for the stored sproc skip, because the stored sproc needs to skip any calling of pytest assume.

@sfc-gh-jdu can you take a look and unify the two changes? you can either revert the other change @sfc-gh-lmukhopadhyay has add, or move your change to where @sfc-gh-lmukhopadhyay has add. just keep in mind that the stored sproc skip needs to make sure no calling of method that requries pytest assume.

Reply:
I have to manually add this flag here https://github.com/snowflakedb/snowpark-python/pull/1626/files#diff-01c804bbf883a36af718bc99184689135c2eef596fba42a5f4f5ded2e025ef16R271, which means there should be another PR to add it in regression test file.

Reply:
hmm, i am not sure if that is going to work for the stored sproc, I believe this line https://github.com/snowflakedb/snowpark-python/pull/1721/files#diff-745c288d4dfae7e02324d3c313319c2a5757068eaa059e8a9dd08698e458b7d9R384 requires the pytest_assume package.

if you set no_check here, will you be able to skip that line? also i don't think it is necessary to check IS_IN_STORED_PROC, if the flag can be used, we can probably change in our regression test run to make sure this flag is set to false.

Reply:
I see, no I don't think we can skip this line. Then we still need to add the flag in regression test @sfc-gh-lmukhopadhyay. Let me revert this change

Reply:
@sfc-gh-jdu sorry, what i mean is do you need this flag to be set inside the sql_counter initializer, can you simply do it at the same place as where the stored sproc is skipped?

Reply:
that place only skips `sql_count_check` decorator, but it seems not skip SqlCounter if it is created directly

Reply:
There should be another PR to add that flag in regression test file if you don't want to check IS_IN_STORED_PROC here, though it works

Reply:
This looks good to me, thanks for reverting the stored_sproc change!

Reply:
The current change looks good. @sfc-gh-lmukhopadhyay @sfc-gh-azhan let's follow up to simply reuse the SKIP_SQL_COUNT_CHECK in the stored sproc test infra, so that we do not have to do a special check for IS_IN_STORED_PROC. @sfc-gh-jdu can you create a ticket for this?

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-1462284 thank y'all!

Reply:
we can just pass the flag as a pytest argument when we need it. cc @sfc-gh-lmukhopadhyay